### PR TITLE
Added Form validation and input handle in the form

### PR DESCRIPTION
**Pull request by divya-prav**
Used useForm hook to handle the form input fields
Added validation for required fields
Added form submission function(can see the data in the console).

### CODE REVIEW

File: src/app/edit/page.tsx
Comment (johnjang94) [inline src/app/edit/page.tsx:1]: It looks like you hard-coded report object. Are you planning to change the code to dynamic later?

**Review by johnjang94 — COMMENTED** [review]
I have left a question on specific part 😁 Please have a look.

File: src/app/edit/page.tsx
Comment (divya-prav) [reply src/app/edit/page.tsx:1]: Yes, I am planning to make it dynamic later and  I am not sure if the report API routes are ready yet.

Comment (johnjang94) [reply src/app/edit/page.tsx:1]: OK, thank you for your confirmation. I will go ahead and merge your code for now. Thank you!

**Pull request merged**